docs/community/software/mesh-metrics.mdx

```diff
@@ -0,0 +1,16 @@
+---
+id: community-mesh-metrics
+title: Mesh-metrics
+sidebar_label: Mesh-metrics
+sidebar_position: 3
+---
+
+:::note
+
+This is a community project maintained by @cordelster.
+Development can be followed on [GitHub](https://github.com/cordelster/mesh-metrics).
+Support should be sought from the respective authors.
+
+:::
+
+Mesh-metrics is a script that uses the meshtastic-cli and a device connected by usb to send metrics-requests from a list of devices and output the results in a format to be ingested by node_exporter to populate a Prometheus/VictoriaMetrics TDB. Along with the script, a Grafana dashboard and some alert examples are included. Mesh-metrics was created and is maintained by cordel and more information on its use and setup can be found on the [Mesh-metrics Github page](https://github.com/cordelster/mesh-metrics)
```

Comment:
is VictoriaMetrics one or two words?  What does TBD mean in this case?

Reply:
I have seen "Victoria Metrics" as two words by the host project, though naming in package managers vary.
TDB (time series database).

I can link the home pages for Prometheus and Victoria Metrics if you like, and make it Victoria Metrics to follow suit with upstream. Just tell me what you want to see and it will be done.

Reply:
Looks aweseome!